Snabble/UI/Utilities/UIView+RestrictDynamicTypeSize.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  UIView+RestrictDynamicTypeSize.swift
+//  Snabble
+//
+//  Created by Anastasia Mishur on 26.05.22.
+//
+
+import UIKit
+
+extension UIView {
+    func restrictDynamicTypeSize(from min: UIContentSizeCategory?, to max: UIContentSizeCategory?) {
```

Comment:
If we don't have an else option. I'd like to mark the function sorry for the misunderstanding and use an explicit use in code otherwise t

```swift
if #available(iOS 15, *) {
   ....restrictDynamicTypeSize....
}
```

Reply:
changed myself 3bab7f778917aa2aa4b252f90a7f15a551a15466